uniffi_macros/src/util.rs

```diff
@@ -151,11 +151,11 @@ pub fn parse_comma_separated<T: UniffiAttributeArgs>(input: ParseStream<'_>) ->
 }
 
 #[derive(Default)]
-pub struct ArgumentNotAllowedHere;
+struct ArgumentNotAllowedHere;
 
 impl Parse for ArgumentNotAllowedHere {
-    fn parse(input: ParseStream<'_>) -> syn::Result<Self> {
-        parse_comma_separated(input)
+    fn parse(_input: ParseStream<'_>) -> syn::Result<Self> {
+        unreachable!();
```

Comment:
Huh?

Reply:
It is unreachable in practice, so I figured it makes sense to call it out as such. I only noticed that because when I was debugging I got a little confused and though this might have been implicated. I would have been less confused with `unreachable()`. I'm fine putting this back though if you think it has value.